install-build-deps.sh

```diff
@@ -12,18 +12,16 @@ else
   OSARCH="linux-amd64"
 fi
 
-
+go install github.com/gohugoio/hugo
```

Comment:
@martina-if this actually installs it without SASS support, which requires CGO... Seems like we aren't using SASS, but I don't really know.

Reply:
We are using SCSS